src/containerapp/azext_containerapp/custom.py

```diff
@@ -3258,3 +3260,54 @@ def set_registry_job(cmd, name, resource_group_name, server, username=None, pass
     containerapp_job_registry_set_decorator.construct_payload()
     r = containerapp_job_registry_set_decorator.set()
     return r
+
+
+# maintenance config
+def add_maintenance_config(cmd, resource_group_name, env_name, duration, start_hour_utc, weekday):
```

Comment:
We should have argument validate logic to validate if the input argument are valid, right?